client/src/js/services/InventoryService.js

```diff
@@ -38,6 +39,13 @@ function InventoryService($http, util) {
       .then(util.unwrapHttpResponse);
   }
 
+  /** get inventory metadata report */
+  function receipt(filetype) {
```

Comment:
This probably shouldn't be called "receipt".   Maybe "report()"?  It isn't actually a receipt, it is a report of all items in the inventory, right?